test/markdown-parser.js

```diff
@@ -229,4 +229,43 @@ code
 
     expect(tokens).to.deep.equal([]);
   });
+
```

Comment:
Could you move these tests to below the other front matter-related tests please?